lib/cog/commands/group.ex

```diff
@@ -2,22 +2,33 @@ defmodule Cog.Commands.Group do
   @moduledoc """
   This command allows the user to manage groups of users.
 
-  ## Usage
-
-      group --create <groupname>
-      group --drop <groupname>
-      group --add --user=<username> <groupname>
-      group --remove --user=<username> <groupname>
-      group --list
-
-  ## Example
-
-      @bot operable:group --create ops
-      @bot operable:group --create engineering
-      @bot operable:group --add --user=bob ops
-      @bot operable:group --remove --user=bob ops
-      @bot operable:group --drop ops
-      @bot operable:group --list
+  USAGE:
```

Comment:
Super nitpick mode. I think the others exclude the `:`. Might be nice to stay consistent.